internal/admission/defaulting_test.go

```diff
@@ -111,3 +111,77 @@ func TestTimeout(t *testing.T) {
 		require.InDelta(t, timeout.Seconds(), time.Since(start).Seconds(), 0.1, "timeout duration is not respected")
 	})
 }
+
+func TestStrictMode(t *testing.T) {
+	tests := []struct {
```

Comment:
Can we move them closer to the testCase loop?

Reply:
It is gone, tests were moved to functions.